pkg/skaffold/debug/transform_go.go

```diff
@@ -155,7 +163,7 @@ func extractDlvSpec(args []string) *dlvSpec {
 		return nil
 	}
 	// delve's defaults
-	spec := dlvSpec{apiVersion: 2, log: false, headless: false}
+	spec := dlvSpec{apiVersion: 2, log: false, headless: false, wait: true}
```

Comment:
This is changing the default behavior. Are you sure about that? This could instead by false by default and if someone wants to have wait as true, then they'd need to have the label or annotation

Reply:
This has to start from `true` as the default behaviour of `dlv` is also to wait, this changes with the `--continue` arg which makes the wait to `false` ([here](https://github.com/pkoutsovasilis/skaffold/blob/a1bd2365ce9a6f40d92e44dea3935816ea8001fa/pkg/skaffold/debug/transform_go.go#L179)) and the default behaviour of dlv. I can rename the `wait` to `continue` and have it start at false?!